pkg/manager/manager.go

```diff
@@ -111,191 +89,122 @@ func (m *Manager) setTargetGenerator(targetGen func() southbound.TargetIf) {
 	southbound.TargetGenerator = targetGen
 }
 
-func (m *Manager) initializeStores() {
+// Run starts a synchronizer based on the devices and the northbound services.
+func (m *Manager) Run() {
+	log.Info("Starting Manager")
+
+	if err := m.Start(); err != nil {
+		log.Fatal("Unable to run Manager", err)
+	}
+}
+
+// Start starts the manager
+func (m *Manager) Start() error {
 	opts, err := certs.HandleCertPaths(m.Config.CAPath, m.Config.KeyPath, m.Config.CertPath, true)
 	if err != nil {
 		log.Fatal(err)
```

Comment:
We should return error and log here. We already call Fatal when we call m.Start()